src/AppBundle/Subscriber/BugLabelNewIssueSubscriber.php

```diff
@@ -36,15 +36,15 @@ public function onIssues(GitHubEvent $event)
         }
 
         $responseData = array('issue' => $issueNumber = $data['issue']['number']);
-        // Ignore non-bugs or issue which already has a status
+        // Ignore non-bugs or issue which already has a statu
         if ('bug' !== strtolower($data['label']['name']) || null !== $this->statusApi->getIssueStatus($issueNumber, $repository)) {
             $responseData['status_change'] = null;
             $event->setResponseData($responseData);
 
             return;
         }
 
-        $newStatus = Status::NEEDS_REVIEW;
+        $newStatus = Status::UNCONFIRMED;
```

Comment:
I didn't see neither the reviewed status used as "good to close" nor a confirmed bug closed due to lack of solutions (they stay open for ever ^^), but in this case I think you're right. 👍